inc/class-shortcake-bakery.php

```diff
@@ -117,4 +120,56 @@ public static function get_shortcake_admin_dependencies() {
 		return $r;
 	}
 
+	public function action_admin_enqueue_scripts() {
+		wp_enqueue_script( 'shortcake-bakery-admin', plugin_dir_url( dirname( __FILE__ ) ) . 'assets/js/shortcake-bakery-admin.js', array( 'media-views', 'shortcode-ui' ) );
+		wp_enqueue_style( 'shortcake-bakery', plugin_dir_url( dirname( __FILE__ ) ) . 'assets/css/shortcake-bakery.css' );
+
+		$strings = array(
+			'text' => array(
+				'addEmbed' => __( 'Insert Embed Code', 'shortcake-bakery' ),
+				'insertButton' => __( 'Insert embed', 'shortcake-bakery' ),
+				'customEmbedLabel' => __( 'Paste any custom embed code here. If it matches a known post element, that post element will be used rather than the embed code.', 'shortcake-bakery' ),
+				'noReversalMatches' => __( 'The embed code provided doesn\'t match any known post elements. This means that it may not display as expected.', 'shortcake-bakery' ),
```

Comment:
Sure. They're all set in the view using `jQuery.text()`, so I didn't think it was necessary, but it's still good practice.